File: benchmarks/suite/string_build.go
package main
import ("fmt";"strings";"time")
func main(){ t0:=time.Now(); n:=400000; var sb strings.Builder
  for i:=0;i<n;i++{ sb.WriteString("abcdefghijklmnopqrstuvwxyz") }
  el:=time.Since(t0).Seconds(); fmt.Println(sb.Len()); fmt.Printf("elapsed: %.6fs\n",el) }

File: benchmarks/suite/string_build.rs
use std::time::Instant;
fn main(){ let t0=Instant::now(); let n=400000; let mut s=String::with_capacity(64);
  for _ in 0..n { s.push_str("abcdefghijklmnopqrstuvwxyz"); }
  let el=t0.elapsed().as_secs_f64(); println!("{}",s.len()); println!("elapsed: {:.6}s",el); }
